gaia-fast-list.js

```diff
@@ -264,12 +329,20 @@ Internal.prototype = {
     header.classList.add('gfl-header');
     section.appendChild(header);
     section.appendChild(background);
+    section.classList.add('gfl-section');
 
     return section;
   },
 
   populateItem: function(el, i) {
-    poplar.populate(el, this.getRecordAt(i));
+    // debug('populate item', el, i);
+    var record = this.getRecordAt(i);
+    if (!poplar.populate(el, record)) {
```

Comment:
nit: not a huge fan of having an important action (populate) tucked away in a condition.
I'm guilty of this too, but I wouldn't mind the extra `poplarResult` var. (or a better name)